### PR TITLE
Generalized adapter names in all topics/examples (removed notes)

### PR DESCRIPTION
**Pull request by TinaUll**
Hi Rainier, it would be great if you could take a look to make sure that I did not miss anything for the generalizing the adapter names. I removed the notes too and deleted the big example in one of topics and just added a reference to the adapter specific documentation. Thank you!

### CODE REVIEW

File: V1.3/Administration/delete-an-adapter-component.md [hunk 1]
```diff
@@ -16,10 +16,10 @@ Complete the following steps to delete an adapter component:
 
       Example using `curl` :
 
-    **Delete OpcUa1 adapter component**
+    **Delete an adapter component**
 
       ```bash
-      curl -X DELETE "http://localhost:5590/api/v1/configuration/system/components/OpcUa1"
+      curl -X DELETE "http://localhost:5590/api/v1/configuration/system/components/<componentId>"
```
Comment (rmuolic) [inline V1.3/Administration/delete-an-adapter-component.md:22]: Line 41 mentions Modbus, this should be removed correct? @TinaUll

Comment (rmuolic) [inline V1.3/Administration/delete-an-adapter-component.md:22]: <ComponentId>

File: V1.3/Administration/start-and-stop-ingress-component.md [hunk 2]
```diff
@@ -11,16 +11,16 @@ To control data ingress, you can start and stop the ingress components of an ada
 Complete the following steps to start an individual ingress component:
 
 1. Use any of the [Configuration tools](xref:ConfigurationTools1-3) capable of making HTTP requests.
-2. Run a `POST` command to the following endpoint, replacing `<ingressComponentId>` with the ingress component that you want to start: `http://localhost:5590/api/v1/administration/<ingressComponentId>/Start`
+2. Run a `POST` command to the following endpoint, replacing `<componentId>` with the ingress component that you want to start: `http://localhost:5590/api/v1/administration/<componentId>/Start`
```
Comment (rmuolic) [inline V1.3/Administration/start-and-stop-ingress-component.md:14]: Change componentId to ComponentID

File: V1.3/Administration/start-and-stop-ingress-component.md [hunk 3]
```diff
@@ -11,16 +11,16 @@ To control data ingress, you can start and stop the ingress components of an ada
...
     **Note:** `5590` is the default port number. If you selected a different port number, replace it with that value.
 
     Example using `curl`:
 
-    **Start the OpcUa1 ingress component**
+    **Start the adapter ingress component**
 
     ```bash
-    curl -d "" -X POST "http://localhost:5590/api/v1/Administration/OpcUa1/Start"
+    curl -d "" -X POST "http://localhost:5590/api/v1/Administration/<componentId>/Start"
```
Comment (rmuolic) [inline V1.3/Administration/start-and-stop-ingress-component.md:23]: Change componentId to ComponentId

File: V1.3/Administration/start-and-stop-ingress-component.md [hunk 4]
```diff
@@ -29,14 +29,14 @@ Complete the following steps to stop an individual ingress component:
...
     **Note:** `5590` is the default port number. If you selected a different port number, replace it with that value.
 
     Example using `curl`:
 
-    **Stop the Modbus1 ingress component**
+    **Stop the adapter ingress component**
 
     ```bash
-    curl -d "" -X POST "http://localhost:5590/api/v1/Administration/Modbus1/Stop"
+    curl -d "" -X POST "http://localhost:5590/api/v1/Administration/<componentId>/Stop"
```
Comment (rmuolic) [inline V1.3/Administration/start-and-stop-ingress-component.md:41]: Change componentId to ComponentId

File: V1.3/Configuration/discovery-configuration.md [hunk 5]
```diff
@@ -71,7 +70,7 @@ The following example shows the status of all discoveries. The discovery id in t
         "itemsFound": 4,
         "newItems": 0,
         "newAssets": 0,
-        "resultUri": "http://127.0.0.1:5590/api/v1/Configuration/OpcUa1/Discoveries/8ff855f1-a636-490a-bb31-207410a6e607/result",
+        "resultUri": "http://127.0.0.1:5590/api/v1/Configuration/<componentId>/Discoveries/8ff855f1-a636-490a-bb31-207410a6e607/result",
```
Comment (rmuolic) [inline V1.3/Configuration/discovery-configuration.md:73]: Change componentId to ComponentId

File: V1.3/Configuration/discovery-configuration.md [hunk 6]
```diff
@@ -95,4 +94,4 @@ The following example shows the status of all discoveries. The discovery id in t
 | api/v1/configuration/_componentId_/dataselection/select?discoveryid=_discoveryId_     | POST      | Adds the discovered items to data selection with selected set to `true`                                                                   |
 | api/v1/configuration/_componentId_/dataselection/unselect?discoveryid=_discoveryId_   | POST      | Adds the discovered items to data selection with selected set to `false`
 
-**Note:** Replace _componentId_ with the Id of your adapter component, for example OpcUa1.<br>Replace _discoveryId_ with the Id of the discovery for which you want to perform the action.
+**Note:** Replace _componentId_ with the Id of your adapter component.<br>Replace _discoveryId_ with the Id of the discovery for which you want to perform the action.
```
Comment (rmuolic) [inline V1.3/Configuration/discovery-configuration.md:97]: Change componentId to ComponentId

File: V1.3/Configuration/system-components-configuration.md [hunk 7]
```diff
@@ -60,7 +58,7 @@ Complete the following steps to add a new component to the system:
         **Note:** Run this command from the same directory where the file is located.
 
         ```bash
-        curl -d "@AddComponent.json" -H "Content-Type: application/json" -X PUT "http://localhost:5590/api/v1/configuration/system/components/Modbus1"
+        curl -d "@AddComponent.json" -H "Content-Type: application/json" -X PUT "http://localhost:5590/api/v1/configuration/system/components/<componentId>"
```
Comment (rmuolic) [inline V1.3/Configuration/system-components-configuration.md:61]: Change componentId to ComponentId

File: V1.3/Configuration/system-components-configuration.md [hunk 8]
```diff
@@ -102,10 +100,10 @@ Complete the following steps to delete a specific existing component:
     Example using `curl`:
 
     **Note:** Run this command from the same directory where the file is located. <br>
-    *Delete OpcUa1 component*
+    *Delete adapter component*
   
     ```bash
-    curl -X DELETE "http://localhost:5590/api/v1/configuration/system/components/OpcUa1/"
+    curl -X DELETE "http://localhost:5590/api/v1/configuration/system/components/<componentId>/"
```
Comment (rmuolic) [inline V1.3/Configuration/system-components-configuration.md:106]: Change componentId to ComponentId

File: V1.4/Administration/start-and-stop-ingress-component.md [hunk 9]
```diff
@@ -11,16 +11,16 @@ To control data ingress, you can start and stop the ingress components of an ada
 Complete the following steps to start an individual ingress component:
 
 1. Use any of the [Configuration tools](xref:ConfigurationTools1-4) capable of making HTTP requests.
-2. Run a `POST` command to the following endpoint, replacing `<ingressComponentId>` with the ingress component that you want to start: `http://localhost:5590/api/v1/administration/<ingressComponentId>/Start`
+2. Run a `POST` command to the following endpoint, replacing `<componentId>` with the ingress component that you want to start: `http://localhost:5590/api/v1/administration/<componentId>/Start`
```
Comment (rmuolic) [inline V1.4/Administration/start-and-stop-ingress-component.md:14]: Change componentId to ComponentId

File: V1.4/Administration/start-and-stop-ingress-component.md [hunk 10]
```diff
@@ -11,16 +11,16 @@ To control data ingress, you can start and stop the ingress components of an ada
...
     **Note:** `5590` is the default port number. If you selected a different port number, replace it with that value.
 
     Example using `curl`:
 
-    **Start the OpcUa1 ingress component**
+    **Start the adapter ingress component**
 
     ```bash
-    curl -d "" -X POST "http://localhost:5590/api/v1/Administration/OpcUa1/Start"
+    curl -d "" -X POST "http://localhost:5590/api/v1/Administration/<componentId>/Start"
```
Comment (rmuolic) [inline V1.4/Administration/start-and-stop-ingress-component.md:23]: Change componentId to ComponentId

File: V1.4/Administration/start-and-stop-ingress-component.md [hunk 11]
```diff
@@ -29,14 +29,14 @@ Complete the following steps to stop an individual ingress component:
 
 1. Start any configuration tool capable of making HTTP requests.
 
-2. Run a `POST` command to the following endpoint, replacing `<ingressComponentId>` with the ingress component that you want to stop: `http://localhost:5590/api/v1/administration/<ingressComponentId>/Stop`
+2. Run a `POST` command to the following endpoint, replacing `<componentId>` with the ingress component that you want to stop: `http://localhost:5590/api/v1/administration/<componentId>/Stop`
```
Comment (rmuolic) [inline V1.4/Administration/start-and-stop-ingress-component.md:32]: Change componentId to ComponentId

File: V1.4/Administration/start-and-stop-ingress-component.md [hunk 12]
```diff
@@ -29,14 +29,14 @@ Complete the following steps to stop an individual ingress component:
...
     **Note:** `5590` is the default port number. If you selected a different port number, replace it with that value.
 
     Example using `curl`:
 
-    **Stop the Modbus1 ingress component**
+    **Stop the adapter ingress component**
 
     ```bash
-    curl -d "" -X POST "http://localhost:5590/api/v1/Administration/Modbus1/Stop"
+    curl -d "" -X POST "http://localhost:5590/api/v1/Administration/<componentId>/Stop"
```
Comment (rmuolic) [inline V1.4/Administration/start-and-stop-ingress-component.md:41]: Change componentId to ComponentId

File: V1.4/Configuration/discovery-configuration.md [hunk 13]
```diff
@@ -71,7 +70,7 @@ The following example shows the status of all discoveries. The discovery id in t
         "itemsFound": 4,
         "newItems": 0,
         "newAssets": 0,
-        "resultUri": "http://127.0.0.1:5590/api/v1/Configuration/OpcUa1/Discoveries/8ff855f1-a636-490a-bb31-207410a6e607/result",
+        "resultUri": "http://127.0.0.1:5590/api/v1/Configuration/<componentId>/Discoveries/8ff855f1-a636-490a-bb31-207410a6e607/result",
```
Comment (rmuolic) [inline V1.4/Configuration/discovery-configuration.md:73]: Change componentId to ComponentId

File: V1.4/Configuration/discovery-configuration.md [hunk 14]
```diff
@@ -95,4 +94,4 @@ The following example shows the status of all discoveries. The discovery id in t
 | api/v1/configuration/_componentId_/dataselection/select?discoveryid=_discoveryId_     | POST      | Adds the discovered items to data selection with selected set to `true`                                                                   |
 | api/v1/configuration/_componentId_/dataselection/unselect?discoveryid=_discoveryId_   | POST      | Adds the discovered items to data selection with selected set to `false`
 
-**Note:** Replace _componentId_ with the Id of your adapter component, for example OpcUa1.<br>Replace _discoveryId_ with the Id of the discovery for which you want to perform the action.
+**Note:** Replace _componentId_ with the Id of your adapter component.<br>Replace _discoveryId_ with the Id of the discovery for which you want to perform the action.
```
Comment (rmuolic) [inline V1.4/Configuration/discovery-configuration.md:97]: Change componentId to ComponentId

File: V1.4/Configuration/system-components-configuration.md [hunk 15]
```diff
@@ -60,7 +58,7 @@ Complete the following steps to add a new component to the system:
         **Note:** Run this command from the same directory where the file is located.
 
         ```bash
-        curl -d "@AddComponent.json" -H "Content-Type: application/json" -X PUT "http://localhost:5590/api/v1/configuration/system/components/Modbus1"
+        curl -d "@AddComponent.json" -H "Content-Type: application/json" -X PUT "http://localhost:5590/api/v1/configuration/system/components/<componentId>"
```
Comment (rmuolic) [inline V1.4/Configuration/system-components-configuration.md:61]: Change componentId to ComponentId

File: V1.4/Configuration/system-components-configuration.md [hunk 16]
```diff
@@ -102,10 +100,10 @@ Complete the following steps to delete a specific existing component:
     Example using `curl`:
 
     **Note:** Run this command from the same directory where the file is located. <br>
-    *Delete OpcUa1 component*
+    *Delete adapter component*
   
     ```bash
-    curl -X DELETE "http://localhost:5590/api/v1/configuration/system/components/OpcUa1/"
+    curl -X DELETE "http://localhost:5590/api/v1/configuration/system/components/<componentId>/"
```
Comment (rmuolic) [inline V1.4/Configuration/system-components-configuration.md:106]: Change componentId to ComponentId

File: V1/Administration/delete-an-adapter-component.md [hunk 17]
```diff
@@ -16,10 +16,10 @@ Complete the following steps to delete an adapter component:
 
       Example using `curl` :
 
-    **Delete OpcUa1 adapter component**
+    **Delete an adapter component**
 
       ```bash
-      curl -X DELETE "http://localhost:5590/api/v1/configuration/system/components/OpcUa1"
+      curl -X DELETE "http://localhost:5590/api/v1/configuration/system/components/<componentId>"
```
Comment (rmuolic) [inline V1/Administration/delete-an-adapter-component.md:22]: Change componentId to ComponentId

File: V1/Administration/start-and-stop-an-adapter.md [hunk 18]
```diff
@@ -24,20 +22,18 @@ Complete the procedure appropriate for your operating system to start or stop an
 
     Example:
 
-    _Start_ PI Adapter for OPC UA
+    _Start_ PI Adapter for \<AdapterName\>
 
     ```cmdline
-    sudo systemctl start pi.adapter.opcua
+    sudo systemctl start pi.adapter.<adapterName>
```
Comment (rmuolic) [inline V1/Administration/start-and-stop-an-adapter.md:28]: AdapterName

File: V1/Administration/start-and-stop-an-adapter.md [hunk 19]
```diff
@@ -24,20 +22,18 @@ Complete the procedure appropriate for your operating system to start or stop an
...
     ```
 
     Example:
 
-    _Stop_ PI Adapter for Modbus TCP
+    _Stop_ PI Adapter for \<AdapterName\>
   
       ```cmdline
-      sudo systemctl stop pi.adapter.modbus
+      sudo systemctl stop pi.adapter.<adapterName>
```
Comment (rmuolic) [inline V1/Administration/start-and-stop-an-adapter.md:36]: AdapterName

File: V1/Administration/start-and-stop-ingress-component.md [hunk 20]
```diff
@@ -11,16 +11,16 @@ To control data ingress, you can start and stop the ingress components of an ada
 Complete the following steps to start an individual ingress component:
 
 1. Use any of the [Configuration tools](xref:ConfigurationTools) capable of making HTTP requests.
-2. Run a `POST` command to the following endpoint, replacing `<ingressComponentId>` with the ingress component that you want to start: `http://localhost:5590/api/v1/administration/<ingressComponentId>/Start`
+2. Run a `POST` command to the following endpoint, replacing `<componentId>` with the ingress component that you want to start: `http://localhost:5590/api/v1/administration/<componentId>/Start`
```
Comment (rmuolic) [inline V1/Administration/start-and-stop-ingress-component.md:14]: Change componentId to ComponentId

File: V1/Administration/start-and-stop-ingress-component.md [hunk 21]
```diff
@@ -29,14 +29,14 @@ Complete the following steps to stop an individual ingress component:
 
 1. Start any configuration tool capable of making HTTP requests.
 
-2. Run a `POST` command to the following endpoint, replacing `<ingressComponentId>` with the ingress component that you want to stop: `http://localhost:5590/api/v1/administration/<ingressComponentId>/Stop`
+2. Run a `POST` command to the following endpoint, replacing `<componentId>` with the ingress component that you want to stop: `http://localhost:5590/api/v1/administration/<componentId>/Stop`
```
Comment (rmuolic) [inline V1/Administration/start-and-stop-ingress-component.md:32]: Change componentId to ComponentId

File: V1/Administration/start-and-stop-ingress-component.md [hunk 22]
```diff
@@ -29,14 +29,14 @@ Complete the following steps to stop an individual ingress component:
...
     **Note:** `5590` is the default port number. If you selected a different port number, replace it with that value.
 
     Example using `curl`:
 
-    **Stop the Modbus1 ingress component**
+    **Stop the adapter ingress component**
 
     ```bash
-    curl -d "" -X POST "http://localhost:5590/api/v1/Administration/Modbus1/Stop"
+    curl -d "" -X POST "http://localhost:5590/api/v1/Administration/<ComponentId>/Stop"
```
Comment (rmuolic) [inline V1/Administration/start-and-stop-ingress-component.md:41]: Change componentId to ComponentId

File: V1/Configuration/system-components-configuration.md [hunk 23]
```diff
@@ -60,7 +58,7 @@ Complete the following steps to add a new component to the system:
         **Note:** Run this command from the same directory where the file is located.
 
         ```bash
-        curl -d "@AddComponent.json" -H "Content-Type: application/json" -X PUT "http://localhost:5590/api/v1/configuration/system/components/Modbus1"
+        curl -d "@AddComponent.json" -H "Content-Type: application/json" -X PUT "http://localhost:5590/api/v1/configuration/system/components/<componentId>"
```
Comment (rmuolic) [inline V1/Configuration/system-components-configuration.md:61]: Change componentId to ComponentId

File: V1/Configuration/system-components-configuration.md [hunk 24]
```diff
@@ -102,10 +100,10 @@ Complete the following steps to delete a specific existing component:
     Example using `curl`:
 
     **Note:** Run this command from the same directory where the file is located. <br>
-    *Delete OpcUa1 component*
+    *Delete adapter component*
   
     ```bash
-    curl -X DELETE "http://localhost:5590/api/v1/configuration/system/components/OpcUa1/"
+    curl -X DELETE "http://localhost:5590/api/v1/configuration/system/components/<componentId>/"
```
Comment (rmuolic) [inline V1/Configuration/system-components-configuration.md:106]: Change componentId to ComponentId

File: V1.4/Administration/delete-an-adapter-component.md [hunk 25]
```diff
@@ -16,10 +16,10 @@ Complete the following steps to delete an adapter component:
 
       Example using `curl` :
 
-    **Delete OpcUa1 adapter component**
+    **Delete an adapter component**
 
       ```bash
-      curl -X DELETE "http://localhost:5590/api/v1/configuration/system/components/OpcUa1"
+      curl -X DELETE "http://localhost:5590/api/v1/configuration/system/components/<componentId>"
```
Comment (rmuolic) [inline V1.4/Administration/delete-an-adapter-component.md:22]: Change componentId to ComponentId

**Review by rmuolic — COMMENTED** [review]
Minor comments as noted in the Conversation. Throughout, I would change componentId to ComponentId for consistency.
 
.../Administration/delete-an-adapter-component.md, Line 41 mentions Modbus that perhaps should be removed.

File: V1/Administration/start-and-stop-an-adapter.md (hunk 18 above)
Comment (TinaUll) [reply V1/Administration/start-and-stop-an-adapter.md:28]: Keeping it lowercase because the whole command is lowercase

**Review by rmuolic — APPROVED** [review]

**Pull request merged**